reactive-streams/src/main/scala/fs2/interop/reactivestreams/StreamSubscriber.scala

```diff
@@ -102,19 +106,23 @@ object StreamSubscriber {
     def onFinalize: F[Unit]
 
     /** producer for downstream */
-    def dequeue1: F[Either[Throwable, Option[A]]]
+    def dequeue1: F[Either[Throwable, Option[Chunk[A]]]]
 
     /** downstream stream */
-    def stream(subscribe: F[Unit])(implicit ev: ApplicativeError[F, Throwable]): Stream[F, A] =
+    def stream(
+        subscribe: F[Unit]
+    )(implicit ev: ApplicativeError[F, Throwable]): Stream[F, Chunk[A]] =
```

Comment:
Change return here to `Stream[F, A]`

Reply:
https://github.com/typelevel/fs2/pull/2666/commits/d99c684026159103000af81b482f4c6de25fc6c3